src/Columns/ColumnVector.h

```diff
@@ -105,7 +111,10 @@ class ColumnVector final : public COWHelper<ColumnVectorHelper, ColumnVector<T>>
 
 public:
     using ValueType = T;
-    using Container = PaddedPODArray<ValueType>;
+    static constexpr bool is_POD = !is_big_int_v<T>;
+    using Container = std::conditional_t<is_POD,
+                                         PaddedPODArray<ValueType>,
+                                         std::vector<ValueType>>;
```

Comment:
ColumnArray::sizeAt(n) may pass -1. It's safe for PaddedPODArray since it always allocate some unused bytes at the begin and end of the array. However it's dangerous for std::vector.